test/test_helper.rb

```diff
@@ -286,6 +286,23 @@ def refute_authorizes(user, record, action)
     end
   end
 
+  def assert_association(user, record, association, _policy_class)
+    %w[create attach detach destroy edit].each do |action|
+      refute_authorizes(user, record, :"#{action}_#{association}?")
+    end
+
+    begin
+      @authorization_client.authorize(user, record, :avo_show?, policy_class: policy_class)
+
+      assert_authorizes(user, record, :"view_#{association}?")
+    rescue Avo::NotAuthorizedError
+      refute_authorizes(user, record, :"view_#{association}?")
+    end
+
+    # TODO: I'm not clear on what `record` is used in show_association?
```

Comment:
My intention here was to use the passed policy to assert that it refers to that policy, but I'm unclear on how that works.

Is it really getting passed the "object" of the association as the record or are we just looking up the same policy as self because it's not a different record? That would imply that record is not a RubyGem in the RubyGemPolicy when we plan on checking show_association?